collector/kernel/CMakeLists.txt

```diff
@@ -217,6 +217,7 @@ lint_shell_script_bundle(
   kernel-collector-scripts
   SOURCES
     entrypoint.sh
+    entrypoint-kct.sh
```

Comment:
Move it to `kernel_collector_test_docker` directory and just call it `entrypoint.sh`?